frontend/pages/hosts/details/cards/About/About.tsx

```diff
@@ -137,6 +137,20 @@ const About = ({
     );
   };
 
+  const renderBattery = () => {
+    if (typeof aboutData.batteries !== "object") {
+      return null;
+    }
+    return (
+      <div className="info-grid__block">
+        <span className="info-grid__header">Battery</span>
+        <span className="info-grid__data">
+          {aboutData.batteries[0]?.health}
```

Comment:
Can `batteries` be null? If so, I think you'd want to use `aboutData.batteries?.[0]?.health`

Reply:
Not individual batteries in the array, just the `batteries` value itself can be null.

Reply:
But an interface is definitely needed to tighten this up!

Reply:
Also `typeof null` evaluates to `'object'`